.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,76 @@
+name: unit-tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+      - "open-release/*.master"
+  
+jobs:
+  run-tests:
+    if: (github.repository != 'openedx/edx-platform' && github.repository != 'edx/edx-platform-private') || (github.repository == 'openedx/edx-platform' && (startsWith(github.base_ref, 'open-release') == true))
```

Comment:
Do we need this conditional? It seems to me like it will never be true for our case. If this job is intended to run, we should remove the conditional. If it's never expected to be run, we should remove the job. Seeing as it's the only job in here, it seems we should remove the conditional.

Reply:
I Agree.

Reply:
Yep, it's not the `edx-platform` repository. We should be able to remove this line.